src/components/EnterpriseApp/index.jsx

```diff
@@ -1,24 +1,68 @@
 import React from 'react';
-import { Switch, Route, Redirect } from 'react-router-dom';
 import PropTypes from 'prop-types';
+import { Switch, Route, Redirect } from 'react-router-dom';
+import MediaQuery from 'react-responsive';
 
 import AdminPage from '../../containers/AdminPage';
 import CodeManagementPage from '../../containers/CodeManagementPage';
 import RequestCodesPage from '../../containers/RequestCodesPage';
+import Sidebar from '../../containers/Sidebar';
 import SupportPage from '../SupportPage';
 import NotFoundPage from '../NotFoundPage';
 import ErrorPage from '../ErrorPage';
 
-import { removeTrailingSlash } from '../../utils';
+import { breakpoints, removeTrailingSlash } from '../../utils';
 import { features } from '../../config';
 
+import './EnterpriseApp.scss';
+
 class EnterpriseApp extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.contentWrapperRef = React.createRef();
+    this.sidebarRef = null;
+
+    this.state = {
+      sidebarWidth: null,
+    };
+  }
+
   componentDidMount() {
     const {
       match: { params: { enterpriseSlug } },
     } = this.props;
 
     this.props.fetchPortalConfiguration(enterpriseSlug);
+    this.props.toggleSidebarToggle(); // ensure sidebar toggle button is in header
```

Comment:
We contain the sidebar component within `EnterpriseApp` such that it will only appear for routes within the `EnterpriseApp` component. However, without this, the `Header` component is not aware of when the `SidebarToggle` should appear. By having dispatching this `toggleSidebarToggle` action on mount/unmount, we can make sure the `SidebarToggle` appears only for `EnterpriseApp`. Let me know if you would like additional clarification.